pypeit/core/coadd1d.py

```diff
@@ -0,0 +1,2100 @@
+import os
+import scipy
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy import stats
+from astropy.io import fits
+from astropy import convolution
+from IPython import embed
+from astropy.table import Table
+
+from pkg_resources import resource_filename
+from pypeit import utils
+from pypeit import msgs
+from pypeit.core import load, save
+from pypeit.core.wavecal import wvutils
+from pypeit.core import pydl
+from astropy import constants as const
+c_kms = const.c.to('km/s').value
```

Comment:
```
from astropy import constants
```
or
```
import astropy.constants
```
and then define and use this only inside the function where you need it.  Please don't define global variables.

Reply:
Done.